daemon/src/routes_maker.rs

```diff
@@ -186,6 +187,31 @@ pub async fn post_cfd_action(
     Ok(status::Accepted(None))
 }
 
+#[derive(Debug, Clone, Deserialize)]
+pub struct WalletRecoverRequest {
+    pub seed_words: String,
+}
+
+#[rocket::post("/wallet/recover", data = "<wallet_recover_request>")]
+pub async fn post_wallet_recover(
+    wallet_recover_request: Json<WalletRecoverRequest>,
+    wallet_actor: &State<Wallet>,
```

Comment:
Instead of doing this, I would prefer if we go towards https://github.com/itchysats/itchysats/issues/774.

Reply:
I will make a new PR to address #774 and rebase this onto it